src/terraform.ts

```diff
@@ -42,8 +42,26 @@ interface ModuleProvidersResponse {
     [provider: string]: string;
   };
 }
+
+export interface TerraformInfoResponse {
+  v: number;
+  required_version: string;
+  discovered_version: string;
```

Comment:
`discovered_version` might be optional. If a user has no Terraform installation, the response looks like this:

```json
{"v": 0, "required_version": "~> 1.2"}
```

Reply:
Good catch